blockbase/sass/blocks/_post-comments.scss

```
@@ -89,7 +89,7 @@
 					display: inline-block;
 					line-height: calc( var(--wp--custom--form--checkbox--unchecked--sizing--height) + 2 * var(--wp--custom--form--border--width));
 					margin-left: 0;
-					padding-left: 3em;
+					padding-left: calc( var(--wp--custom--form--checkbox--unchecked--sizing--width) + var(--wp--custom--gap--baseline) );
```

```suggestion
					padding-left: calc( var(--wp--custom--form--checkbox--unchecked--sizing--width) + var(--wp--custom--gap--baseline) / 1.5 ) );
```

Let's decrease this slightly to 1/3 of its actual amount (10px). Otherwise we're good to go.